policyengine_us/parameters/gov/states/co/ccap/re_determination/re_determination_hhs_smi_rate.yaml

```diff
@@ -0,0 +1,12 @@
+description: Colorado limits its Child Care Assistance Program re-determination to applicants with adjusted gross income lower than this rate of household size state median income.
```

Comment:
the description might be too long and cause the test to fail

Reply:
I tried a shorter one, but it also failed.

Reply:
I shorten the description once again, and it didn't pass.